ui/src/holdingpen-new/containers/DetailPageContainer/AuthorDetailPageContainer.tsx

```diff
@@ -177,11 +177,15 @@ const AuthorDetailPageContainer: React.FC<AuthorDetailPageContainerProps> = ({
                       </Col>
                     </Row>
                   </CollapsableForm.Section>
-                  {author?.get('_error_msg') && (
+                  {author?.get('status') === 'error' && (
                     <CollapsableForm.Section header="Errors" key="errors">
-                      <div className="bg-waiting error-code">
-                        {author?.get('_error_msg')}
-                      </div>
+                      <p>
+                        See error details here:{' '}
+                        <a
+                          href={`https://workflows.inspire.dev.siscern.org//dagrun/list/?_flt_0_run_id=${id}`}
+                          target="_blank"
+                        >{`https://workflows.inspire.dev.siscern.org//dagrun/list/?_flt_0_run_id=${id}`}</a>
```

Comment:
could we please make it configurable? Do we have other places that the url is hardcoded?

Reply:
Done 👍 that's the only hardcoded url